src/components/App.css

```diff
@@ -175,6 +175,16 @@ input[type="number"] {
 }
 
 .battle-toolbar--button {
+  padding: 4px;
+  height: 40px;
+  margin-top: 25px;
```

Comment:
Previously the spacing of the dropdown items was provided by these rules on `battle-toolbar--options-dropdown`:

```
display: flex;
justify-content: space-between;
flex-direction: column;
```

It looks like if we go with a larger `margin-top` instead the above rules are no longer necessary (I removed them and it looked OK!) I think the `width` of `.battle-toolbar--options-container` plus the `margin-top` gets them to display vertically. If this seems good we could remove these extra styles to simplify things a bit.